Tests/XcodeprojTests/GenerateXcodeprojTests.swift

```diff
@@ -43,8 +43,34 @@ class GenerateXcodeprojTests: XCTestCase {
         }
 #endif
     }
-    
+
+    func testXcconfigOverrideValidatesPath() {
+#if os(macOS)
```

Comment:
I have a marginal preference we let this run on Linux, where it should work, just to keep as much of the code as possible cross-platform consistent.